### ISSUE TITLE
Change hotkeys for balloon jumping

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#147]
Balloon hotkeys for balloon have changed. The should be 

JS jumps close to Sweater.
JW jumps to Wall.
JN jumps to near Wall.
JC jumps to center of Play Area.

**Comment by jessegreenberg** [comment]
Fixeed in above commit. But i noticed that the balloon can be dragged up/down while jumping to wall or sweater.  This should be fixed.

**Comment by jessegreenberg** [comment]
It also looks like the balloon is trying to snap beyond the wall when the all is visible

**Comment by jessegreenberg** [comment]
Completed and fixed in the commit above.  Also fixed issue where balloon would jump too far to the right.  Closing.
